src/app/components/VerticalList/index.tsx

```diff
@@ -0,0 +1,10 @@
+import { COLORS } from 'styles/theme/colors'
+import { styled } from '@mui/material/styles'
+import Box from '@mui/material/Box'
+
+export const VerticalList = styled(Box)(({ theme }) => ({
```

Comment:
Great, this could also be used for `ProposalDetailView`. (It's already on master branch, you'll see it if you rebase this.)

Reply:
I will enable proposal list in a next PR. For now we show that list in Dashboard card only